frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -305,6 +333,7 @@ export default class NewForm extends Vue {
   @Prop() initialSubmissionsGap!: number;
   @Prop({ default: '' }) initialTitle!: string;
   @Prop({ default: null }) initialWindowLength!: null | number;
+  @Prop({ default: '' }) invalidParameterName!: string;
```

Comment:
```suggestion
  @Prop({ default: null }) invalidParameterName!: null | string;
```

preferimos usar `null` para indicar que una cadena no existe en vez de usar `''`. esto se conoce como ["valores mágicos"](https://en.wikipedia.org/wiki/Magic_string) (o ["sentinelas"](https://en.wikipedia.org/wiki/Sentinel_value)). Esos sentinelas eran útiles cuando era imprescindible poder usar un mismo tipo de datos para los valores válidos y los inválidos (onda, en C donde se tenía que poder expresar arreglos y cadenas sin pasar explícitamente la longitud, entonces se tenía que encodificar dentro de la cadena de alguna manera). Los lenguajes modernos ya permiten especificar otros tipos de datos para evitar el uso de sentinelas tradicionales. En este caso, `null` es un valor que es imposible que una cadena válida tome, entonces sabemos que cuando este campo es `null`, es porque ningún parámetro es inválido.

igual en `contest/new.ts` hay que usar `null`